examples/x86_64.swiftinterface

```diff
@@ -0,0 +1,511 @@
+  public init(from decoder: Swift.Decoder) throws
+  final public func encode(to encoder: Swift.Encoder) throws
+  @objc deinit
+}
+public struct SendChatMessageRequest : Swift.Codable {
+  public let content: Swift.String
+  public let senderDisplayName: Swift.String?
+  public let type: AzureCommunicationChat.ChatMessageType?
+  public init(content: Swift.String, senderDisplayName: Swift.String? = nil, type: AzureCommunicationChat.ChatMessageType? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct CreateChatThreadRequest : Swift.Codable {
+  public let topic: Swift.String
+  public let participants: [AzureCommunicationChat.ChatParticipant]?
+  public init(topic: Swift.String, participants: [AzureCommunicationChat.ChatParticipant]? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+@_hasMissingDesignatedInitializers public class SignalingChatParticipant {
+  public var id: AzureCommunicationCommon.CommunicationIdentifier?
+  public var displayName: Swift.String?
+  public var shareHistoryTime: AzureCore.Iso8601Date?
+  @objc deinit
+}
+@_hasMissingDesignatedInitializers public class SignalingChatThreadProperties {
+  public var topic: Swift.String
+  @objc deinit
+}
+@_hasMissingDesignatedInitializers public class BaseChatEvent {
+  public var threadId: Swift.String
+  public var sender: AzureCommunicationCommon.CommunicationIdentifier?
+  public var senderDisplayName: Swift.String?
+  public var recipient: AzureCommunicationCommon.CommunicationIdentifier?
+  @objc deinit
+}
+@_hasMissingDesignatedInitializers public class BaseChatThreadEvent {
+  public var threadId: Swift.String
+  public var version: Swift.String
+  @objc deinit
+}
+@_hasMissingDesignatedInitializers public class BaseChatMessageEvent : AzureCommunicationChat.BaseChatEvent {
+  public var id: Swift.String
+  public var createdOn: AzureCore.Iso8601Date?
+  public var version: Swift.String
+  public var type: AzureCommunicationChat.ChatMessageType
+  @objc deinit
+}
+@_hasMissingDesignatedInitializers public class ChatMessageReceivedEvent : AzureCommunicationChat.BaseChatMessageEvent {
+  public var message: Swift.String
+  @objc deinit
+}
+@_hasMissingDesignatedInitializers public class ChatMessageEditedEvent : AzureCommunicationChat.BaseChatMessageEvent {
+  public var message: Swift.String
+  public var editedOn: AzureCore.Iso8601Date?
+  @objc deinit
+}
+@_hasMissingDesignatedInitializers public class ChatMessageDeletedEvent : AzureCommunicationChat.BaseChatMessageEvent {
+  public var deletedOn: AzureCore.Iso8601Date?
+  @objc deinit
+}
+@_hasMissingDesignatedInitializers public class TypingIndicatorReceivedEvent : AzureCommunicationChat.BaseChatEvent {
```

Comment:
It seems all of these Event classes should also be `struct`s, unless there is some specific reason they are `class`es.